pyuvsim/simsetup.py

```diff
@@ -1144,23 +1162,36 @@ def initialize_uvdata_from_keywords(yaml_filename=None, antenna_layout_filename=
 
     extra_kwds = {k: v for k, v in six.iteritems(kwargs) if k in valid_param_names}
 
+    # Strangely enough, the initialize_uvdata_from_params doesn't set the polarizations.
+    # This is probably because they were previously hardcoded into the complete_uvdata
+    # function. However, it doesn't really make sense to put them there, so we put
+    # them here.
```

Comment:
It was a legacy of the original design. `pyuvsim` was originally written to read in a uvfits file and simulate based on that, and ignored the polarization array. The `init_uvdata_out` function was made to set up the output file, and was written long before the parameter option was put in. In a much earlier version such simulations were deprecated and now it's exclusively parameter based, so it makes sense to re-organize.

Reply:
OK, going to remove most of these comments then :-) (ce6c7e0b973cdc8235f465573d7deb7b469d05d4)